### PR TITLE
Use lein figwheel plugin for easier fruit cutting

### PR DESCRIPTION
**Pull request by ryanhinkel**
I'm really a fan of using this plugin.

### CODE REVIEW

**Comment by kliph** [comment]
@ryanhinkel watch out! I've got a plan.

giphy

File: README.org [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Procedurally generated sunrises and sunsets
 
 * To start
-=(figwheel-up)=
```
Comment (kliph) [inline README.org]: I think we should leave both of these.  I still start it by using `dev.repl> (figwheel-up)` after invoking `cider-jack-in` in a clojure file in the project.

**Review by kliph — CHANGES_REQUESTED** [review]
Please leave the instructions for starting the CLJS REPL from within the `dev-repl` namespace. 💯

Other than that I tested it with the bumped dependencies and it works.  I'll see about getting `lein-figwheel` into the template and deploying it.

**Comment by ryanhinkel** [comment]
Ok, I added back and described both approaches of standing up the figwheel dev environment.

**Review by kliph — APPROVED** [review]

**Pull request merged**